### DISCUSSION TITLE
@use 'igniteui-angular/theming' as *;

### DISCUSSION THREAD

**Discussion by mikerentmeister** [IgniteUI/igniteui-angular#11184] [Q&A]
I'm currently using version 13.0.11 of this IgniteUi. After updating from the trial version of this package (igniteui-angular) to the licensed (@ingfragistics/igniteui-angular), I am now unable to build my application. I get an error from the Sass Loader saying:

```
Error: Module build failed (from ./node_modules/sass-loader/dist/cjs.js):
SassError: Can't find stylesheet to import.
   ╷
11 │ @use 'igniteui-angular/theming' as *;
   │ ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   ╵
  libs\common\styles\main.scss 11:1  root stylesheet
```

In my angular.json file I have the stylePreprocessorOptions set to this:
```
"stylePreprocessorOptions": {
              "includePaths": [
                "node_modules/"
              ]
            }
```

In my tsconfig paths I have:
`"igniteui-angular": ["./node_modules/@infragistics/igniteui-angular"]`

Does anyone know why I might be getting this error off-hand?

**Comment by Lipata** [comment]
@mikerentmeister I guess you've followed the [Ignite UI Licensing](https://www.infragistics.com/products/ignite-ui-angular/angular/components/general/ignite-ui-licensing) topic. Did you try the suggestion there:

```
"stylePreprocessorOptions": {
        "includePaths": [
            "node_modules/@infragistics"
        ]
    }
```

**Comment by mikerentmeister** [comment]
Yeah I tried that too. What ended up working for me in the end was doing this instead:

`@use '@infragistics/igniteui-angular/theming' as *;`
